### PR TITLE
feat(style): makes `background-clip` to support multiple values

### PR DESCRIPTION
**Pull request by stalniy**
Relates to #2441

### CODE REVIEW

File: documentation/md/style.md [hunk 1]
```diff
@@ -300,7 +300,7 @@ A background image may be applied to a node's body.  The following properties su
 
 The following properties apply to all images of a node:
 
- * **`background-clip`** : How background image clipping is handled; may be `node` for clipped to node shape or `none` for no clipping.
+ * **`background-clip`** : How background image clipping is handled; may be `node` for clipped to node shape or `none` for no clipping. Supports multiple values, so some `background-image`s can be clipped and others not (useful when you want to draw a background image outside node)
```
Comment (maxkfranz) [inline documentation/md/style.md]:
```suggestion
 * **`background-clip`** : How background image clipping is handled; may be `node` for clipped to node shape or `none` for no clipping. Supports multiple values, so some `background-image`s can be clipped and others not (useful when you want to draw a background image outside node).
```

Comment (maxkfranz) [reply documentation/md/style.md]: Add period at end

Comment (stalniy) [reply documentation/md/style.md]: Done

**Comment by stalniy** [comment]
Awesome! When can I expect this to be released?

**Comment by maxkfranz** [comment]
Patches at the start of the week, features at the start of the month.

https://blog.js.cytoscape.org/2018/12/04/new-release-schedule/

**Pull request merged**